lasagne/tests/layers/test_normalization.py

```diff
@@ -50,8 +50,8 @@ def ground_truth_normalizer(c01b, k, n, alpha, beta):
         for c in range(out.shape[2]):
             for x in range(out.shape[3]):
                 out[:, r, c, x] = ground_truth_normalize_row(
-                        row=c01b[:, r, c, x],
-                        k=k, n=n, alpha=alpha, beta=beta)
+                    row=c01b[:, r, c, x],
+                    k=k, n=n, alpha=alpha, beta=beta)
```

Comment:
I changed this because I got a pep8 failure when running py.test

```
/home/joncrall/code/Lasagne/lasagne/tests/layers/test_normalization.py:53:25: E126 continuation line over-indented for hanging indent
                        row=c01b[:, r, c, x],
```

I'll try undoing that and committing. Maybe Travis CI doesn't have that check enabled.